tower-http/src/services/fs/serve_dir/mod.rs

```diff
@@ -180,6 +180,23 @@ impl<F> ServeDir<F> {
         self
     }
 
+    /// Informs the service that it should also look for a precompressed zstd
+    /// version of _any_ file in the directory.
+    ///
+    /// Assuming the `dir` directory is being served and `dir/foo.txt` is requested,
+    /// a client with an `Accept-Encoding` header that allows the zstd encoding
+    /// will receive the file `dir/foo.txt.zz` instead of `dir/foo.txt`.
```

Comment:
.zstd here instead

Reply:
If we are talking about files, I guess `.zst` is the right choice: https://www.iana.org/assignments/media-types/application/zstd